src/charm.py

```diff
@@ -77,91 +73,91 @@ def configure_workload_container(self, event) -> None:
         """Configure/set up workload container inside pod."""
         missing = self._missing_charm_configs()
         if missing:
-            msg = 'Required config(s) empty: {}'.format(', '.join(sorted(missing)))
+            msg = f"Required config(s) empty: {', '.join(sorted(missing))}"
```

Comment:
I'd return the sorter configs in `_missing_charm_configs` instead of sorting them later